pkg/thing/interactors/update_data.go

```diff
@@ -77,9 +84,9 @@ func validateSchema(data entities.Data, configList []entities.Config) bool {
 // ValidateSchemaNumber validates the value received against its type defined in the sensor's schema
 func ValidateSchemaNumber(value float64, valueType int) bool {
 	switch valueType {
-	case 1: // int
+	case intType:
 		return value >= math.MinInt32 && value <= math.MaxInt32
-	case 2: // float
+	case floatType:
 		return true
 	case 5: // int64
```

Comment:
Can you create const for this number too?

Reply:
Yes. Fixed.